migrations/1658221793207-MaterializedView.ts

```diff
@@ -0,0 +1,48 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class MaterializedView1658221793207 implements MigrationInterface {
+  name = "MaterializedView1658221793207";
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`drop materialized view if exists deposits_mv`);
+    await queryRunner.query(`
+      create materialized view deposits_mv as
+        select
+          d."depositId",
+          d."depositTxHash",
+          d."sourceChainId",
+          d."destinationChainId",
+          d.amount,
+          t.symbol,
+          t.decimals,
+          d."depositorAddr",
+          d."referralAddress",
+          d."depositDate",
+          hmp.usd as "tokenUsdPrice",
+          (d."realizedLpFeePctCapped" / power(10, 18)) * (d.amount / power(10, t.decimals)) * hmp.usd as "realizedLpFeeUsd",
```

Comment:
`realizedLpFeePctCapped` doesn't include `relayerFeePct` yet does it?

Reply:
@nicholaspai not yet. The bridge fee re-calculation is coming as a separate PR